client/k8s/k8s.go

```diff
@@ -44,7 +43,6 @@ func newRawClientConfig(config Config) *rest.Config {
 		Username:        config.Username,
 		Password:        config.Password,
 		BearerToken:     config.BearerToken,
-		Insecure:        config.Insecure,
```

Comment:
`rest.Config` does not seem to have a field insecure anymore. Is this a problem for you guys?